### ISSUE TITLE
MQTT - stand alone reflector

### ISSUE THREAD

**Issue by mitra42** [mitra42/frugal-iot#27]
For a network of sensors to operate independently of backhaul, a single e.g. ESP32 should be able to manage local broker plus sending on up to main server.

**Comment by mitra42** [comment]
There are a number of MQTT broker libraries I have found. 
https://github.com/terrorsl/sMQTTBroker looks good - does Retain and QOS 0,1
https://github.com/mlesniew/PicoMQTT does not do retain
https://github.com/alexCajas/EmbeddedMqttBroker looks like dependent on ESP32 environment, but could be misreading

**Comment by mitra42** [comment]
Duplicate of #40
